### ISSUE TITLE
Implement new NFT metadata

### ISSUE THREAD

**Issue by psiemens** [onflow/kitty-items#166]
We want to update Kitty Items to implement the new NFT metadata standard (ref: https://github.com/onflow/flow-nft/pull/52).

This work has been started in the `metadata` branch.

Remaining TODOs:
- Store images in IPFS and return them as part of the metadata
  - @psiemens to share more info here
- Define a new custom metadata view for Kitty Items (e.g. `ItemData`)
- Update the following scripts to return NFT data from metadata views:
  - https://github.com/onflow/kitty-items/blob/metadata/cadence/scripts/kittyItems/get_kitty_item.cdc
  - Get account item (from JS web app): https://github.com/onflow/kitty-items/blob/metadata/web/src/flow/script.get-account-item.js
  - Get listing item (from JS web app): https://github.com/onflow/kitty-items/blob/metadata/web/src/flow/script.get-listing.js

**Comment by psiemens** [comment]
@joshuahannan and @10thfloor: I created an [NFT Storage](https://nft.storage/) account for the Flow team. We can use it to pin Kitty Items image assets to IPFS. Will share the details offline.

The images that need to be pinned are all here: https://github.com/onflow/kitty-items/tree/master/web/public/images/kitty-items

The approach I'm thinking:
- We pre-pin all of the images from that directory
- Create a dictionary field on the `KittyItems` contract that maps each kind/rarity combination to the correct IPFS hash

I just realized that we have multiple sizes for each image, which is a must-have for good frontend performance. However, the [current image views being discussed](https://github.com/onflow/flow-nft/pull/55) only specify the single image. I think we should prototype some ways to multiple image sizes in this repo's copy of `MetadataViews` and then bring those to that PR.
